lib/helper/Protractor.js

```diff
@@ -224,6 +224,24 @@ class Protractor extends SeleniumWebdriver {
   }
 
   /**
+   * {{> ../webapi/waitForVisibleThenClick }}
+   */
+  waitForVisibleThenClick(locators, sec, delay = 0) {
```

Comment:
I think you can remove that from Protractor as it already inherits this method for SeleniumWebdriverJS